test/index.js

```diff
@@ -6,6 +6,60 @@ const { parseArgs } = require('../index.js');
 
 // Test results are as we expect
 
+test('when short option used as flag then stored as flag', function(t) {
+  const passedArgs = ['-f'];
+  const expected = { flags: { f: true }, values: { f: undefined }, positionals: [] };
+  const args = parseArgs(passedArgs);
+
+  t.deepEqual(args, expected);
+
+  t.end();
+});
+
+test('when short option withValue used with value then stored as value', function(t) {
+  const passedArgs = ['-f', 'bar'];
+  const passedOptions = { withValue: ['f'] };
```

Comment:
What happens in this case if you don't specify `withValue`? Would `bar` end up as a positional?

Reply:
Correct.

Reply:
It might be worth adding a test for this.

Reply:
This case is tested in `test('arg is true and positional is identified', function(t) {` _but_ that test also has a gratuitous use as both a flag and with a value so not clean. 

Happy to add a new test.

(I have been going light on the tests to date because the naming convention itself is still under discussion so potential churn coming, and only a single test file. Interested in getting wider and more methodical coverage.)

Reply:
Added